sdk/core/Azure.Core/src/AccessToken.cs

```diff
@@ -63,7 +85,7 @@ public override bool Equals(object? obj)
         /// <inheritdoc />
         public override int GetHashCode()
         {
-            return HashCodeBuilder.Combine(Token, ExpiresOn);
+            return HashCodeBuilder.Combine(Token, ExpiresOn, TokenType);
```

Comment:
Same question - could changes to hash codes for the same type break existing apps in any way?

Reply:
They shouldn't, since the app would probably have a consistent reference to some version of Azure.Core. Perhaps if two different apps were trying to compare an instance across apps, but that seems like a very edge case scenario.